SharpPcap/LibPcap/PcapDevice.cs

```diff
@@ -131,7 +131,6 @@ public IntPtr PcapHandle
 
         protected bool Active
         {
-            get { return isActive; }
```

Comment:
If no code is actually is using the value of `Active`, does it make sense for it to exist as a property?

Reply:
I mean we could make it a method right? Not sure that buys us much. I always thought we might need to know if we were active vs. open but I guess not.

Reply:
The only need for this method/property is to record device information such as timestamp type and link layer after device is activated, no ?

Reply:
Yep, I think that's what its doing. We call it a handful of places in the code.

Reply:
@kayoub5 it sounds like its ok to use a property rather than a method according to this doc, https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ms229054(v=vs.100)?redirectedfrom=MSDN

Reply:
This changes the value of TimstampType and LinkType, I would consider that to fall into "side effects", thus better being implemented as a method.

Reply:
Yeah maybe it can go either way. I'm ok if we leave it though. The ms link above says updating internal caches etc isn't sufficient, this is close to what we are doing there. If it somehow improves clarity then we should consider it but it doesn't bug me to leave it for now.